src/appleseed/foundation/image/exrimagefilewriter.cpp

```diff
@@ -41,6 +41,10 @@
 
 // OpenEXR headers.
 #include "foundation/platform/_beginexrheaders.h"
+#if __GNUC__ >= 7
```

Comment:
You can put all pragmas to `"foundation/platform/_beginexrheaders.h"` once, as I mentioned above.
If you do this, we don't need to repeat these pragmas every time.